### PR TITLE
ChoiceFormField of type "select" could be "disabled"

### PR DESCRIPTION
**Pull request by bouland**
Hi,

New PR to add tests from the closed PR #15952 

| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php [hunk 1]
```diff
@@ -59,6 +59,10 @@ public function hasValue()
      */
     public function isDisabled()
     {
+        if (parent::isDisabled() && 'radio' !== $this->type) {
```
Comment (xabbuh) [inline src/Symfony/Component/DomCrawler/Field/ChoiceFormField.php]: should be `'select' === $this->type` (the test won't work for radio inputs)

**Comment by xabbuh** [comment]
Status: Needs work

**Comment by xabbuh** [comment]
:+1: LGTM

ping @jakzal 

Status: Reviewed

**Comment by fabpot** [comment]
Thank you @bouland.

**Pull request closed without merging**